listenbrainz/webserver/templates/profile/selecttimezone.html

```diff
@@ -1,35 +1,14 @@
-{% extends 'base.html' %}
+{% extends 'base-react.html' %}
 
 {% block title %}Select Timezone- ListenBrainz{% endblock %}
 
-{% block content %}
-  <h3 class="page-title">Select Timezone</h3>
-  <p>
-    Your current timezone setting is <b>{{ user_timezone }}</b>. 
-    <br>By choosing your local time zone, 
-    you will have a local timestamps part of your submitted listens. 
-    This also informs as when to generate daily playlists and other recommendations for you.
-  </p>
+{% block profile_content %}
```

Comment:
No need for the profile_content block because base-react defines one already. This one is only needed in user/* template files because template structure is different there.